build.gradle

```diff
@@ -114,13 +118,18 @@ artifacts {
 }
 
 dependencies {
-    testCompile 'junit:junit:4.12'
-    testCompile 'org.mockito:mockito-core:1.10.19'
+    testImplementation 'junit:junit:4.12'
+    testImplementation 'org.mockito:mockito-core:2.19.0'
 
-    compile 'com.android.support:appcompat-v7:26.1.0'
-    compile 'com.android.support:design:26.1.0'
+    implementation 'androidx.appcompat:appcompat:1.1.0'
+    implementation 'com.google.android.material:material:1.0.0'
 
-    androidTestCompile('com.android.support.test.espresso:espresso-core:2.2.2', {
+    implementation('com.android.support:appcompat-v7:28.0.0', {
+        exclude group: 'androidx.appcompat'
```

Comment:
My guess is that these `exclude` lines are what allows `com.android.support:appcompat-v7` to be used along side `androidx.appcompat:appcompat`, right?

Reply:
Yes. You suggested it on the Friday.